.github/workflows/widget_test.yaml

```diff
@@ -0,0 +1,19 @@
+name: Unit_Test
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - master
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
```

Comment:
Looks like fetch-depth defaults to 1. Can you remove this line?

https://github.com/actions/checkout